
Currently Loaded Modules:
  1) tools/miniconda/python3.8/4.9.2

 

WARNING: overwriting environment variables set in the machine
overwriting variable nnUNet_raw_data_base nnUNet_preprocessed RESULTS_FOLDER OUTPUT
  Running command git clone -q https://github.com/FabianIsensee/hiddenlayer.git /tmp/pip-install-zp_nu9hk/hiddenlayer_091682c42b7b482d9f9881ef83f97784
  Running command git checkout -b more_plotted_details --track origin/more_plotted_details
  Switched to a new branch 'more_plotted_details'
  Branch 'more_plotted_details' set up to track remote branch 'more_plotted_details' from 'origin'.
Traceback (most recent call last):
  File "/home/smaijer/.conda/envs/nn/bin/nnUNet_train", line 33, in <module>
    sys.exit(load_entry_point('nnunet', 'console_scripts', 'nnUNet_train')())
  File "/home/smaijer/nnUNet/nnunet/run/run_training.py", line 164, in main
    trainer.initialize(not validation_only)
  File "/home/smaijer/nnUNet/nnunet/training/network_training/nnUNetTrainerV2.py", line 124, in initialize
    self.initialize_network()
  File "/home/smaijer/nnUNet/nnunet/training/network_training/nnUNetTrainerV2.py", line 159, in initialize_network
    self.network = Generic_UNet(self.num_input_channels, self.base_num_features, self.num_classes,
  File "/home/smaijer/nnUNet/nnunet/network_architecture/generic_UNet.py", line 537, in __init__
    self.apply(weightInitializer)
  File "/home/smaijer/.conda/envs/nn/lib/python3.9/site-packages/torch/nn/modules/module.py", line 667, in apply
    module.apply(fn)
  File "/home/smaijer/.conda/envs/nn/lib/python3.9/site-packages/torch/nn/modules/module.py", line 667, in apply
    module.apply(fn)
  File "/home/smaijer/.conda/envs/nn/lib/python3.9/site-packages/torch/nn/modules/module.py", line 667, in apply
    module.apply(fn)
  [Previous line repeated 988 more times]
  File "/home/smaijer/.conda/envs/nn/lib/python3.9/site-packages/torch/nn/modules/module.py", line 666, in apply
    for module in self.children():
  File "/home/smaijer/.conda/envs/nn/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1618, in children
    for name, module in self.named_children():
  File "/home/smaijer/.conda/envs/nn/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1636, in named_children
    for name, module in self._modules.items():
RecursionError: maximum recursion depth exceeded while calling a Python object
